From 271e45339b7b969e98ccef7837cb1b15480b07fc Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 4 Nov 2021 18:58:34 +0000
Subject: [PATCH] unbundle: fix visibility of //build/config/freetype

For system freetype //build/linux:freetype_from_pkgconfig is restricted
to //third_party:freetype_harfbuzz and //third_party/harfbuzz-ng:harfbuzz_source.
However //build/config/freetype:freetype is accessing it too.

Bug: None
Change-Id: Ic4a37b01e4ae221372a9220cbad04c598b844c21
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3259304
Commit-Queue: Thomas Anderson <thomasanderson@chromium.org>
Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/main@{#938388}
---
 build/linux/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/linux/BUILD.gn b/build/linux/BUILD.gn
index 54314c7687..cd49e3d524 100644
--- a/build/linux/BUILD.gn
+++ b/build/linux/BUILD.gn
@@ -23,6 +23,7 @@ if (use_system_freetype) {
   # the system, use with caution,for details see build/config/freetype/BUILD.gn.
   pkg_config("freetype_from_pkgconfig") {
     visibility = [
+      "//build/config/freetype:freetype",
       "//third_party:freetype_harfbuzz",
       "//third_party/harfbuzz-ng:harfbuzz_source",
     ]
